test/turing_model.jl

```diff
@@ -0,0 +1,177 @@
+@testset "turing_model.jl" begin
+    kidiq = CSV.read(joinpath("..", "data", "kidiq.csv"), DataFrame)
+    wells = CSV.read(joinpath("..", "data", "wells.csv"), DataFrame)
+    roaches = CSV.read(joinpath("..", "data", "roaches.csv"), DataFrame)
+    @testset "normal likelihood" begin
+        @testset "standardize=false" begin
+            normal_model = turing_model(@formula(kid_score ~ mom_iq * mom_hs), kidiq)
+            normal_chn = sample(seed!(123), normal_model, NUTS(), MCMCThreads(), 2_000, 2)
+            @test summarystats(normal_chn)[:α, :mean] ≈ 29.30 atol = 2.0
+            @test summarystats(normal_chn)[Symbol("β[1]"), :mean] ≈ 0.533 atol = 0.2
+            @test quantile(normal_chn)[Symbol("β[2]"), Symbol("50.0%")] ≈ 0.593 atol = 0.2
+        end
+
+        @testset "standardize=true" begin
+            normal_model_std = turing_model(
+                @formula(kid_score ~ mom_iq * mom_hs), kidiq; standardize=true
+            )
```

Comment:
```suggestion
            form = @formula(kid_score ~ mom_iq * mom_hs)
            normal_model_std = turing_model(form, kidiq; standardize=true)
```
Easier to read and shorter